packages/react-scripts/scripts/lint.js

```diff
@@ -0,0 +1,22 @@
+// @remove-on-eject-begin
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+// @remove-on-eject-end
+'use strict';
+
+const CLIEngine = require('eslint').CLIEngine;
+
+const cli = new CLIEngine({
+  configFile: require.resolve('eslint-config-react-app'),
```

Comment:
hardcoding this won't respect the changes that user made in `package.json` or `.eslintrc` post eject. IMO we should wrap this line:
```js
// @remove-on-eject-begin
    configFile: require.resolve('eslint-config-react-app'),
// @remove-on-eject-end
```

Reply:
yes ... but then `react-scripts lint` after eject has few errors:

```
/private/tmp/demo20/src/App.js
  1:1   warning  Definition for rule 'jsx-a11y/href-no-hash' was not found  jsx-a11y/href-no-hash
 
/private/tmp/demo20/src/App.test.js
  1:1  warning  Definition for rule 'jsx-a11y/href-no-hash' was not found  jsx-a11y/href-no-hash

/private/tmp/demo20/src/index.js
  1:1  warning  Definition for rule 'jsx-a11y/href-no-hash' was not found  jsx-a11y/href-no-hash

/private/tmp/demo20/src/registerServiceWorker.js
  1:1  warning  Definition for rule 'jsx-a11y/href-no-hash' was not found  jsx-a11y/href-no-hash

✖ 4 problems (0 errors, 4 warnings)
```

Reply:
hmm, do you have `eslint` installed globally / `.eslintrc` file somewhere in the parent folder? i think we need to add `root: true` to `eslintConfig` property in `package.json` post eject. https://eslint.org/docs/user-guide/configuring#using-configuration-files can you try adding it and see if it fix the problem?

Reply:
Sadly, no change after:
```
  "eslintConfig": {
    "root": true,
    "extends": "react-app"
  }
```

Reply:
Also adding `plugins: ['jsx-a11y']` to CLIEngine constructor options doesn't remove the error.

Reply:
maybe running it with `DEBUG=eslint:* yarn lint` would help you found the problem. I cannot reproduce it locally using the same code :(

Reply:
or `DEBUG=eslint:config* yarn lint` if you just want the log related to config